test/PublishCommand.js

```diff
@@ -800,18 +797,18 @@ describe("PublishCommand", () => {
       NODE_ENV: "lerna-test",
     };
 
-    beforeEach((done) => {
-      testDir = initFixture("PublishCommand/normal", done);
-    });
+    beforeEach(() => initFixture("PublishCommand/normal").then((dir) => {
+      testDir = dir;
+
+      // mock out the ENV to a simpler version for testing
+      process.env = mockEnv;
```

Comment:
I moved the `process.env` mocking into the tail-end of `beforeEach` because now (thanks to promises) I can be assured that all fixture initialization that needs the "real" env will not be harmed by this mock.